src/bitcoind.cpp

```diff
@@ -146,19 +142,17 @@ static bool AppInit(int argc, char* argv[])
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #endif
-            tfm::format(std::cout, "Bitcoin server starting\n");
+            tfm::format(std::cout, PACKAGE_NAME "daemon starting\n");
 
             // Daemonize
             if (daemon(1, 0)) { // don't chdir (1), do close FDs (0)
-                tfm::format(std::cerr, "Error: daemon() failed: %s\n", strerror(errno));
-                return false;
+                return InitError(strprintf("daemon() failed: %s\n", strerror(errno)));
```

Comment:
```suggestion
                return InitError(strprintf("%s daemon failed: %s\n", PACKAGE_NAME, strerror(errno)));
```

Reply:
I think it really means that `deamon()` failed, so I'd prefer to keep the `(`,`)`.

Reply:
Agree.